abodepy/helpers/constants.py

```diff
@@ -53,6 +53,8 @@
 
 OAUTH_TOKEN_URL = BASE_URL + 'api/auth2/claims'
 
+PARAMS_URL = BASE_URL + 'api/v1/devices_beta/'
```

Comment:
Is this an endpoint only for changing params, or is it more just a generic endpoint and the 'action' parameter value tells the server what to do? Is this something that will change once abode finished their beta?

Reply:
The only place I've seen it being used is for changing device parameters for some devices (not all). For example, changing which mode a door/window sensor is active seems to also use this endpoint. I'm assuming it will change at some point and we will just have to update it. It doesn't seem to be passed in the device JSON data unfortunately.

Reply:
Just seems like a weird endpoint name. But, also, people tend to suck at naming endpoints so... whatever!